README.rst

```diff
@@ -1,3 +1,11 @@
+.. ATTENTION::
```

Comment:
A very minor comment, 'attention' is not highlighted very well, maybe using a header for it would be better.
<img width="878" alt="Screenshot 2021-07-15 at 17 15 25" src="https://user-images.githubusercontent.com/21034626/125813630-371e2ca1-307b-49b9-9401-c36f65b922aa.png">

Reply:
Strange, Pycharm renders it like this:
![image](https://user-images.githubusercontent.com/20990866/125823043-5f3aa525-23c1-4bfe-876f-892c43fdc5d0.png)
I'll check how I can change it.

Reply:
Apparently Github doesn't support this (github/markup#68). Updated the formatting to something Github can render.

Reply:
Thanks! Looks great.